app/assets/stylesheets/vendors/_foundation_overrides.scss

```diff
@@ -48,7 +48,11 @@ div.has-error > input:focus {
 }
 
 .has-error, .facolor-error, .facolor-warning {
-    /* max-width: 30rem; */
+    max-width: 30rem;
+}
+
+.modal.is-active, .has-error, .facolor-error, .facolor-warning {
```

Comment:
These classes override the above max-width: 30rem;

What you can do is this: 

```
.modal {
    .has-error, .facolor-error, .facolor-warning {
        max-width: 35rem;
    }
}
```

This will apply your new max-width *only* inside a modal.